### ISSUE TITLE
Detection framework and # of synthetic data used for Swin-T/S experiments in Table 3?

### ISSUE THREAD

**Issue by Jamshid-Ganiev** [iamkanghyunchoi/mimiq#2]
HI @iamkanghyunchoi, Thank you for amazing work.

Could you clarify which object detection framework (e.g., Mask RCNN, Cascade Mask RCNN, etc.) was used as the detector head for Swin-T and Swin-S backbone experiments reported in Table 3 (COCO dataset) of the paper? The paper doesn't seem to specify this detail explicitly. 

Additionally, for the same Table 3 experiments, were PSAQ-ViT V1 and V2 baselines also evaluated using 10k synthetic calibration images (same as MimiQ default), or were they run with their own original settings?

**Comment by iamkanghyunchoi** [comment]
Thank you for your interest in our paper.

1. Object detection framework
We use a Mask R-CNN detector with Swin-T and Swin-S backbones. The implementation is based on the MMDetection library. The corresponding model configurations can be found here: https://github.com/open-mmlab/mmdetection/tree/main/configs/swin

2. Number of synthetic images for PSAQ
We follow the original experimental setting provided in the PSAQ paper.

**Comment by Jamshid-Ganiev** [comment]
@iamkanghyunchoi, thank you for a swift reply,
I have one more follow-up question on this.
- In Table 3, does "Real FT" (indicated by the red arrow on the attached image) refer to fine-tuning with 10k real images, or the full COCO training set (120K)? If possible, could you clarify this as well?

<img width="990" height="353" alt="Image" src="https://github.com/user-attachments/assets/da41d5e2-df97-471c-b046-4dd88fb417d9" />

**Comment by iamkanghyunchoi** [comment]
We use the full COCO training set while keeping the batch size, number of training steps, and number of epochs unchanged.

**Comment by Jamshid-Ganiev** [comment]
Thank you very much for the reply.
